core-web/libs/edit-content/src/lib/fields/dot-edit-content-file-field/dot-edit-content-file-field.component.ts

```diff
@@ -73,8 +73,11 @@ export class DotEditContentFileFieldComponent implements ControlValueAccessor, O
     constructor() {
         effect(() => {
             const value = this.store.value();
-            this.onChange(value);
-            this.onTouched();
+
+            if (this.onChange && this.onTouched) {
+                this.onChange(value);
+                this.onTouched();
+            }
```

Comment:
Return early is a good practice to handle `if` statements

```suggestion

            if (!this.onChange && !this.onTouched) {
                return;
            }
            
            this.onChange(value);
            this.onTouched();
```

Reply:
Great suggestion, fixed!